### PR TITLE
PWGHF: remove DCA from HFSelTrack table and use TracksExtended instead

### PR DESCRIPTION
**Pull request by aalkin**
* Remove DCA columns from HFSelTrack
* Add `TracksExtended` in corresponding track joins for HF tasks

This moves track DCA calculation to central  `track-extension`/`alice3-track-extension` tasks.

### CODE REVIEW

File: PWGHF/TableProducer/HFD0CandidateSelector.cxx [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 #include "Framework/runDataProcessing.h"
 #include "Framework/AnalysisTask.h"
+#include "Common/DataModel/TrackSelectionTables.h"
```
Comment (vkucera) [inline PWGHF/TableProducer/HFD0CandidateSelector.cxx:20]: Not needed if already included in `HFSecondaryVertex.h`.

**Review by vkucera — CHANGES_REQUESTED** [review]

File: PWGHF/TableProducer/HFD0CandidateSelector.cxx (hunk 1 above)
Comment (aalkin) [reply PWGHF/TableProducer/HFD0CandidateSelector.cxx]: Removed.

**Comment by vkucera** [comment]
Hi @aalkin , I removed remaining unnecessary includes.
Can you please pull those changes and update `EventFiltering/PWGHF/HFFilter.cxx`?

**Comment by aalkin** [comment]
@vkucera done (had to force push due to rebase conflicts)

**Review by vkucera — APPROVED** [review]

**Comment by vkucera** [comment]
> @vkucera done (had to force push due to rebase conflicts)

Thanks

**Review by mpuccio — APPROVED** [review]

**Pull request merged**